src/fetch/transfer_file.py

```diff
@@ -208,15 +209,14 @@ def _process_native_transfers(
             self.eth_transfers.append(transfer)
 
     def _process_token_transfers(self, cow_redirects: dict[Address, Vouch]) -> None:
+        price_day = self.period.end - timedelta(days=1)
```

Comment:
This change is smuggle in? 

What is the motivation? Would we not get a price for the latest Tuesday, if we execute it on Tuesday? At least it would not be deterministic, right? Hence, it makes sense to me.

Reply:
Because we run the script on Tuesday (the same day as end day) and there aren't known daily prices for the current day.

Reply:
In fact, I could probably add a test demonstrating that the api always errors on date.today().